src/components/asset-list/AssetListHeader.js

```diff
@@ -123,29 +123,30 @@ const AssetListHeader = ({
   }, [accountName, truncated]);
 
   return (
-    <ListHeader
-      contextMenuOptions={contextMenuOptions}
-      isCoinListEdited={isCoinListEdited}
-      isSticky={isSticky}
-      title={title}
-      totalValue={totalValue}
-      {...props}
-    >
-      {!title && (
-        <WalletSelectButton
-          deviceWidth={deviceWidth}
-          onChangeWallet={onChangeWallet}
-          textWidth={textWidth}
-          totalValue={totalValue}
-          truncatedAccountName={truncatedAccountName}
-        />
-      )}
-      {totalValue ? (
-        <H1 align="right" letterSpacing="roundedTight" weight="semibold">
-          {totalValue}
-        </H1>
-      ) : null}
-    </ListHeader>
+    <StickyHeader name={title || 'Balances'}>
```

Comment:
Why is this being defaulted to `'Balances'`?

Reply:
yeah, doesnt make much sense, fixed